fpdf/fpdf.py

```diff
@@ -1974,7 +1974,15 @@ def _out(self, s):
         if(self.state == 2):
             self.pages[self.page]["content"] += (s + "\n")
         else:
-            self.buffer += (s + "\n")
+            if PY3K:
+                if not isinstance(s, bytes):
+                    s = s.encode('latin1')
+                s += b"\n"
```

Comment:
According to the readme, this supports Python 2.5, which I suspect would choke compiling this syntax